viewer/packages/api/src/utilities/Image360Action.ts

```diff
@@ -0,0 +1,13 @@
+/*!
+ * Copyright 2024 Cognite AS
+ */
+
+/**
+ * 360 image action to be used by the CogniteViewer.canDo360Action and do360Action
+ */
+export enum Image360Action {
```

Comment:
As it is related 360 Images, I would suggest to move the file to 360-images package(like viewer/packages/360-images/src/*)

Reply:
I didn't understand the structure. Why is 360Helper class not in that package?

Reply:
Well, as Image360Helper is a bridge between the `360-images` package and `Cognite3DViewer`, it is not in 360-images folder/package.
I would suggest to move all implementation of features on 360 images should be in 360-images like `Image360History` and `Image360Action`

Reply:
Ok.